.travis.yml

```diff
@@ -27,7 +27,7 @@ jobs:
       install:
       - make go-mod-download
       before_script:
-      - docker pull kindest/node:v1.17.2
+      - docker pull kindest/node:v1.18.2
```

Comment:
Should this also be 1.18.3 as well?

Reply:
https://hub.docker.com/r/kindest/node/tags
No tag for 1.18.3